roles/openshift_health_checker/openshift_checks/docker_image_availability.py

```diff
@@ -62,9 +65,15 @@ def run(self, tmp, task_vars):
 
     def required_images(self, task_vars):
         deployment_type = get_var(task_vars, "deployment_type")
+        # FIXME(juanvallejo): we should handle gracefully with a proper error
+        # message when given an unexpected value for `deployment_type`.
```

Comment:
> Hmm.. or.... would it make sense to read from the YAML files that defines the known values?

I think, if possible, this idea would make more sense than maintaining a list of values in one of the checks